blocks/test/fixtures/core__more.html

```diff
@@ -0,0 +1 @@
+<!-- wp:core/more {"text": "Continue reading", "noTeaser": true} /-->
```

Comment:
I couldn't add the invalid unit test because it breaks the test because it's being parsed to freeform instead of more.